src/Parser/Priest/Shadow/Modules/Spells/VoidformGraph.js

```diff
@@ -56,8 +50,9 @@ const VoidformGraph = ({
   const insanityData = [];
   const insanityGeneratedData = [];
   const insanityDrain = [];
-  const initialInsanity = insanityEvents.length > 0 ? insanityEvents[0].classResources[0].amount - (insanityEvents[0].resourceChange * 100) - (insanityEvents[0].waste * 100) : VOIDFORM_MINIMUM_INITIAL_INSANITY;
-
+  const initialInsanity = insanityEvents.length > 0 ?
```

Comment:
This ternary is somewhat hard to follow, how about moving the A and B calculations to their own props?

Reply:
Or an if-statement.